plugins/jenkins/src/components/BuildsPage/lib/CITable/CITable.tsx

```diff
@@ -173,13 +175,48 @@ const generatedColumns: TableColumn[] = [
   {
     title: 'Actions',
     sorting: false,
-    render: (row: Partial<Project>) => (
-      <Tooltip title="Rerun build">
-        <IconButton onClick={row.onRestartClick}>
-          <RetryIcon />
-        </IconButton>
-      </Tooltip>
-    ),
+    render: (row: Partial<Project>) => {
+      const ActionWrapper = () => {
+        const [isLoadingRebuild, setIsLoadingRebuild] = useState(false);
+        const alertApi = useApi(alertApiRef);
+
+        const onRebuild = async () => {
+          if (row.onRestartClick) {
+            setIsLoadingRebuild(true);
+            try {
+              await row.onRestartClick();
+              alertApi.post({
+                message: 'Jenkins re-build has been successfully executed',
+                severity: 'success',
+              });
+            } catch (e) {
+              if (e instanceof ResponseError) {
+                alertApi.post({
+                  message: `Jenkins re-build has been failed. Error: ${e.message}`,
```

Comment:
```suggestion
                  message: `Jenkins re-build has failed. Error: ${e.message}`,
```

been redundant here I think

Reply:
Thanks, removed :)